docs/installation/install.md

```diff
@@ -332,6 +332,8 @@ The main networking flags include:
 
 * `--reserved-subnets`: the list of **private CIDRs to be excluded** from the ones used by Liqo to remap remote clusters in case of address conflicts, as already in use (e.g., the subnet of the cluster nodes).
 The Pod CIDR and the Service CIDR shall not be manually specified, as automatically included in the reserved list.
+* `--enable-metrics`: enable cluster peers' **metrics exposition** (disabled by default).
+Metrics are exposed on a **Liqo-Gateway**'s port (5872 by default) and a **prometheus ServiceMonitor** related resource is created.
```

Comment:
Would move this part to the global section, as potentially more general. Keep the description at a high level (do not mention internal details, such as port numbers), which are left to low-level tuning with helm.